Sources/web3swift/Utils/Hooks/NonceMiddleware.swift

```diff
@@ -20,17 +16,15 @@ extension Web3.Utils {
         public var synchronizationPeriod: TimeInterval = 300.0 // 5 minutes
         var lastSyncTime: Date = Date()
 
-        public func functionToRun() async {
-            guard let w3 = self.web3 else {return}
+        public init() { }
 
+        public func functionToRun() async {
+            guard let w3 = self.web3 else { return }
```

Comment:
Two letter var is probably going to fail swiftlint, just `let web3 = self.web3`?

Reply:
Same for here.